### ISSUE TITLE
Docker workflow not starting for arm images

### ISSUE THREAD

**Issue by francesco-ballarin** [FEniCS/dolfinx#2606]
It seems that since ~~yesterday~~ last week every `arm` step of the docker workflow is not starting due to
```
Requested labels: buildjet-4vcpu-ubuntu-2204-arm
Job defined at: FEniCS/dolfinx/.github/workflows/docker.yml@refs/heads/main
Waiting for a runner to pick up this job...
```
Unless the runner is self-hosted, this could very well be a GitHub actions outage, but I can't find any incident report on the status page.

**Comment by francesco-ballarin** [comment]
To clarify for possible end users reading the issue, the side effect of the `arm` workflow not being able to start is that `dolfinx/dolfinx:nightly` has not been updated nightly during the last week.

**Comment by francesco-ballarin** [comment]
@jhale any idea of what is going wrong there? is there anything I can do to help?

Not having an up to date `dolfinx/dolfinx:nightly` to be run on my CI for the last couple of weeks is making it difficult to ensure that my downstream projects are correctly in sync with dolfinx developments.

Thanks,
Francesco

**Comment by jorgensd** [comment]
> @jhale any idea of what is going wrong there? is there anything I can do to help?
> 
> 
> 
> Not having an up to date `dolfinx/dolfinx:nightly` to be run on my CI for the last couple of weeks is making it difficult to ensure that my downstream projects are correctly in sync with dolfinx developments.
> 
> 
> 
> Thanks,
> 
> Francesco
> 
> 

As a temporary workaround, you can use my custom action:

https://github.com/jorgensd/actions/blob/main/install-dolfinx/action.yaml
on top of the dev-env images, as done in: https://github.com/jorgensd/dolfinx_mpc/blob/58c63d140acc503950b38468ed9db9d2049be350/.github/workflows/build_docs.yml#L13-L23

**Comment by francesco-ballarin** [comment]
Sure, I can always install from source, but if you take a complex workflow like
https://github.com/FEMlium/FEMlium/blob/main/.github/workflows/ci.yml
which also targets libraries different than dolfinx, the temporary workaround amounts to some non trivial modifications of the workflow.

**Comment by francesco-ballarin** [comment]
Will be fixed by #2616

**Comment by michalhabera** [comment]
Workflow now runs thanks to @jhale . We agreed that Docker workflow needs split into base image build (heavy, expensive, trigerred only manually) and end-user images (light, nightly).
